.github/workflows/ci-e2e-badger.yaml

```diff
@@ -40,6 +40,17 @@ jobs:
             STORAGE=badger make jaeger-v2-storage-integration-test
             ;;
         esac
+        
+    - name: Scrape Metrics
```

Comment:
This must be done from the scripts, not the workflow, because once a script is finished it shuts down all child processes